src/collections/_documentation/apm/apm-glossary.md

```diff
@@ -0,0 +1,162 @@
+---
+title: APM Glossary
+sidebar_order: 5
+---
+
+## Clock Skew
+
+If you are collecting transactions from more than a single machine, you will likely encounter **clock skew**. When collecting and comparing transaction events that are gathered from different devices, you may observe timestamps that do not align or events in a trace that don't share a time history. Sentry does not attempt to correct the timestamps in your events. The timestamps displayed in Sentry for transactions and each span retain the original values reported by your application/hosting environment.
+
+While you can reduce clock skew by utilizing Network Time Protocol (NTP) or your cloud provider's clock synchronization services, you may still notice small drifts in your data, as synchronizing clocks on small intervals is challenging.
+
+## Performance Metrics
+
+As part of APM and transaction data, we’re exposing a few new aggregate metrics to help customers better understand the performance of their applications.
+
+### Duration Percentiles
+
+The average, 75%, 95%, and 99% duration, help guide customers in identifying transactions that are slower than the user’s target SLAs.
+
+Our APM data is sampled, so the percentiles we present only represent the data we have. Because of sampling, data ranges, filters, or a low traffic transaction, we will often have a case where data is directionally correct, but not accurate. For example, the average of one number is that number, but that’s not usually what people expect to see.
```

Comment:
```suggestion
Transaction events are sampled, so the percentiles we present within Sentry only represents the data we have. Because of sampling, data ranges, filters, or low volume of transactions, we will often have a case where data is directionally correct, but not accurate. For example, the average of one number is that number, but that’s not usually what you may expect to see.
```